subprojects/core/src/main/java/org/gradle/internal/classpath/InstrumentingClasspathFileTransformer.java

```diff
@@ -290,12 +283,14 @@ protected void processResource(ClasspathBuilder.EntryBuilder builder, ClasspathE
             builder.put(resourceEntry.getName(), resourceEntry.getContent(), resourceEntry.getCompressionMethod());
         }
 
+        protected void finishProcessing(ClasspathBuilder.EntryBuilder builder) throws IOException {}
+
         private boolean isClassFile(ClasspathEntryVisitor.Entry entry) {
             return entry.getName().endsWith(".class");
         }
 
         private boolean isManifest(ClasspathEntryVisitor.Entry entry) {
-            return isManifestName(entry.getName());
+            return JarUtil.isManifestName(entry.getName());
```

Comment:
Ok. I agree that, in certain situations, when the context is not clear or there's ambiguity, qualified names can help better understand the meaning behind a symbol. Often though, they add noise, make lines longer (which can actually make it more painful to read code in the web by requiring horizontal scrolling) and the reading less fluent.

We should come up with guidelines around this otherwise I predict a strong chance of these code changes flip flopping between `Replace static import with qualified access` and `Add static import`.